src/SFA.DAS.Forecasting.AcceptanceTests/Payments/Steps/ProcessPaymentEventSteps.cs

```diff
@@ -148,26 +148,18 @@ public void ThenTheForecastingPaymentServiceShouldStoreThePaymentDeclarationsRec
         {
             WaitForIt(() =>
             {
-                Console.WriteLine($"Looking for Payments. Employer Account Id: {Config.EmployerAccountId}");
-                var parameters = new DynamicParameters();
-                parameters.Add("@employerAccountId", receivningEmployerId, DbType.Int64);
-                parameters.Add("@sendingEmployerAccountId", sendingEmployerId, DbType.Int64);
-                var sql = @"
-                        Select* from Payment 
-                        where employerAccountId = @employerAccountId 
-                        and SendingEmployerAccountId = @sendingEmployerAccountId 
-                        and FundingSource = 2
-                        ";
-                var payments = Connection.Query<PaymentModel>(sql, parameters, commandType: CommandType.Text).ToList();
+                var payments = DataContext.Payments
```

Comment:
This needs to return ToList() or the db will be called to work out the values for Count() and All().